src/os/ui/menu/unitmenu.js

```diff
@@ -79,6 +79,12 @@ os.ui.menu.unit.toggleUnit = function(event) {
   os.unit.UnitManager.getInstance().setSelectedSystem(event.type);
 };
 
+/**
+ * @inheritDoc
+ */
+os.ui.menu.Menu.prototype.supportsToggle = function(el) {
```

Comment:
Replacing the `supportsToggle` function on the prototype will impact any `Menu` created after this code is loaded.

Reply:
I had intended an override (not this cut-n-paste), but that isn't going to work here. Perhaps a function passed in as a constructor option could work better.